build_test.go

```diff
@@ -63,7 +63,7 @@ func testBuild(t *testing.T, context spec.G, it spec.S) {
     stacks = ["some-stack"]
     uri = "some-uri"
     version = "some-dep-version"
-`), 0644)
+`), 0600)
```

Comment:
Everything else looks good to me, but I'm wondering why the file permissions on this file had to change?

Reply:
These aren't necessary to the issue. My linter was complaining that the permissions were too open, which seems reasonable to change.